Cyra/part_1/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(hello_world LANGUAGES CXX)
+add_library(one STATIC src/test.cpp include/random.h)
```

Comment:
in general it is preferred to instead use target_include_directories for setting your include directory that contains your header files.

also why do you have a header here that is empty? there isnt a need for this in this step